src/Compilers/CSharp/Portable/Declarations/DeclarationTreeBuilder.cs

```diff
@@ -742,9 +742,19 @@ private static bool CheckMemberForAttributes(Syntax.InternalSyntax.CSharpSyntaxN
 
                 case SyntaxKind.OperatorDeclaration:
                     anyNonTypeMembers = true;
+
+                    // Member names are exposed via NamedTypeSymbol.MemberNames and are used primarily
+                    // as an acid test to determine whether a more in-depth search of a type is worthwhile.
+                    // We decided that it was reasonable to exclude explicit interface implementations
+                    // from the list of member names.
```

Comment:
Consider duplicating the `Handle in the same way as explicit method implementations` comments, rather than the whole reasoning comment.